src/views/RecIM/components/Forms/ImageOptions/index.jsx

```diff
@@ -83,7 +83,7 @@ const ImageOptions = ({
           IsLogoUpdate: true,
         };
 
-        editActivity(component.ID, activityRequest).then(() => {
+        await editActivity(component.ID, activityRequest).then(() => {
```

Comment:
I am not sure you understand what the await keyword does here. We are doing .then and .catch to handle whether the promise is fulfilled or not. Await is redundant at best here from my current understanding.

Reply:
Switched it, I don't have a clear memory of adding these awaits but it's a good catch